### PR TITLE
Add docs step to CI

### PR DESCRIPTION
**Pull request by kahaaga**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,4 +50,40 @@ jobs:
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+        run:
+          julia --project=docs docs/make.jl
+      - name: Doctests
+        run: |
+          julia --project=docs -e '
+            using Documenter: doctest
+            using TimeseriesSurrogates
+            doctest(TimeseriesSurrogates)'
```
Comment (Datseris) [inline .github/workflows/ci.yml:89]: Kinda useless step, we don't have any doctests in the documentation strings to be performed. And I certainly don't see the usage of them, as we have proper documentation hosted anyways.

**Comment by Datseris** [comment]
@kahaaga if you don't mind, when you are done with this PR I can take over and make it be with Makie.jl instead of the vastly inferior Plots.jl.

**Comment by kahaaga** [comment]
I had forgotten to add CI checks for this repo, but this now works. The docs are [here](https://juliadynamics.github.io/TimeseriesSurrogates.jl/previews/PR84/).

**Comment by kahaaga** [comment]
@Datseris Sure! I just merged this, so that we can get doc previews for the other branches. 

You mean to just port `surroplot` to Makie.jl? If so, go ahead, but do so I haven't used it except in its very earliest stages, where I abandoned it, because it was very cumbersome to use. I guess times have changed? 

Feel free to implement changes either here or in a new branch (perhaps better).

**Comment by Datseris** [comment]
I do all my plotting in Makie.jl now, times has changed drammatically yes :D

**Pull request merged**